src/defines.h

```diff
@@ -26,9 +26,9 @@
 // ================================================
 
 // Set parameters of IMU and board used
-#define IMU IMU_BNO085
-#define SECOND_IMU IMU
-#define BOARD BOARD_SLIMEVR
+#define IMU IMU_BMI323
+#define SECOND_IMU IMU_BMI323
+#define BOARD BOARD_CUSTOM
```

Comment:
Please don't change these, the firmware should always compile for the official trackers.